docs/5.api/rpc/network.md

```diff
@@ -432,9 +432,9 @@ Here is the exhaustive list of the error variants that can be returned by `netwo
 > Queries active validators on the network returning details and the state of validation on the blockchain.
 
 - method: `validators`
-- params: `["block hash"]`, `[block number]`, or `[null]` for the latest block
+- params: `["block hash"]`, `[block number]`, `{"epoch_id": epoch id}`, `{"block_id": block number}`, `{"block_id": block hash}`, or `[null]` for the latest block
```

Comment:
I suggest to be consistent with the quotes.
See `["block hash"]`, but `{"block_id": block hash}`.
Same for `epoch_id`.